src/test/ui/lint/lint-exceeding-bitshifts.rs

```diff
@@ -2,8 +2,8 @@
 //[noopt]compile-flags: -C opt-level=0
 //[opt]compile-flags: -O
 //[opt_with_overflow_checks]compile-flags: -C overflow-checks=on -O
-
 // build-pass
+// ignore-pass (emit codegen-time warnings and verify that they are indeed warnings and not errors)
```

Comment:
Did you mean "emits" and "verifies"? I cannot entirely parse this comment.

Reply:
I copypasted it from other tests.
"[Let the test] emit codegen-time warnings and verify that they are indeed warnings and not errors.", I guess.

Reply:
Okay. I'd still prefer if you could clarify here, I found it unclear if this was asking the reader to do something or the compiler or what.

Reply:
Updated.